geoai/osd.py

```
@@ -0,0 +1,129 @@
+        keras_modules = [keras, tf.keras]
+
+    with _osd_lock:
+        original_getters = []
+        patched_targets = set()
+        for m in keras_modules:
+            if m is None:
+                continue
+            if hasattr(m, "get") and (id(m), "get") not in patched_targets:
+                orig_get = m.get
+
+                def patched_get(identifier, orig=orig_get):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `patched_get`

(ANN202)